constraints.txt

```diff
@@ -1,5 +0,0 @@
-# Constraints that apply to pip installed requirements
```

Comment:
Why remove the constraints file?

Reply:
I was aiming to make it work and TEST codebase with 3.6 and 3.9. Since we don't need explicit file with limitations naming `constraints.txt` because it works without those explicit limitations of libs mentioned in constraints.txt file . Thus, I have added it to the `requirements-test.txt`  file directly.

But, I understood we don't want to disturb the flow of that as of now. I have added a separate flow in  https://github.com/apel/apel/commit/09d1e62db2531ad565deb4ea2f95486e56e908b6.